code/src/components/Album.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react'
+import { Artist } from './Artist'
+
+export const Album = (props) => {
+  const generateDelimiter = (index, arrayLength) => {
+    if (index === arrayLength - 2) {
+      return ' & '
+    } else if (index < arrayLength - 2) {
+      return ', '
+    } 
+    
+    return ''
+  }
+
+  return (
+    <div className="album-card">
+      <div className="album-cover">
+        <a href={props.albumLink}>
+          <img className="album-cover-image" src={props.images[0].url} alt={props.albumName} /> 
+        </a>
+
+          <div className="icons">
+            <img className="icon-heart grow" src="/icons/heart.svg" alt="Heart icon" />
+            <img className="icon-play grow" src="/icons/play.svg" alt="Play icon" />
+            <img className="icon-dots grow" src="/icons/dots.svg" alt="Dots icon" />
+          </div>
+      </div>
+
+      <a className="album-name" href={props.albumLink}>{props.albumName}</a>
```

Comment:
Both artist and album title section does not have a hover effect when I'm on 2 or 1 album per row width.